recipes/quill/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+from conans import ConanFile, CMake, tools
+from conans.model.version import Version
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class QuillConan(ConanFile):
+    name = "quill"
+    description = "C++14 Asynchronous Low Latency Logging Library"
+    homepage = "https://github.com/odygrd/quill/"
+    topics = ("conan", "quill", "logging", "log")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+
```

Comment:
```suggestion
    options = {"fPIC": [True, False]}
    default_options = {"fPIC": True}
```

Reply:
The library should build only as `static` as it is using `thread local` and building it as a shared library would have negative performance implications.

do I still need `fPIC` when I am not offering a `shared` option ?

Reply:
fPIC makes sense for static library only.

Reply:
do we have to delete `fPIC` on windows ?

```
        if self.settings.os == "Windows":
            del self.options.fPIC
```

Reply:
Yes, please.